cypress/e2e/safe-apps/permissions_settings.cy.js

```diff
@@ -45,7 +45,10 @@ describe('Permissions settings tests', () => {
       )
     })
 
-    cy.visit(constants.SEPOLIA_TEST_SAFE_5 + constants.appSettingsUrl, { failOnStatusCode: false })
+    cy.visit(`${constants.appsUrl}?safe=${constants.SEPOLIA_TEST_SAFE_5}`, {
```

Comment:
There should be` constants.appSettingsUrl` used instead of `constants.appsUrl `to navigate to the correct url. Please update it, this will fix the tests @katspaugh